components/views/services/index.tsx

```diff
@@ -0,0 +1,46 @@
+import Button from '@/components/button'
+import ServiceData from './dummydata'
+import { StyledServices } from './Services.styles'
+const ServicesPage = () => {
+    return (
+        <StyledServices>
+            <div className="broad-about">
+                <h1>Titie</h1>
+                <h2>Subtext</h2>
+                <hr />
+                <p>
+                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
+                    Optio doloribus quo explicabo soluta distinctio, odio
+                    deserunt eius deleniti adipisci asperiores enim. Qui quam,
+                    similique assumenda dignissimos corporis repellat
+                    perspiciatis quod odio laboriosam officiis temporibus
+                    dolores omnis debitis. Nihil itaque reiciendis voluptate
+                    beatae nemo quaerat animi impedit. Natus dicta rem odit,
+                    aliquid reprehenderit non nesciunt praesentium repellendus
+                    exercitationem. Amet labore distinctio minus reiciendis
+                    officia eius et laudantium. Voluptas iure officiis quidem
+                    commodi minus dolor illo placeat.
+                </p>
+            </div>
+            <div className="details-about">
+                {ServiceData.map((item, index) => [
+                    <div className="detail-section">
```

Comment:
You can remove this div entirely since we aren't mapping over anything, and the `details-about` is the direct parent element

Reply:
need this map im pretty sure

Reply:
I designed the backend. I know we do not need this map lol